card.py

```diff
@@ -24,3 +24,11 @@ def value_name(self):
 
     def __repr__(self):
         return f'{self.value_name}{self.suit}'
+
+    def __lt__(self, other):
+        return self.value < other.value
+
+    def __radd__(self, other):
+        if self.score + other > 21:
```

Comment:
I'm reasonably confident that this has bugs in it. This would be a great time to get into testing. We should make a test file called `test_player.py` and test `player.score` to make sure it's returning what we think it will.

You'll want several combinations of tests for potential hands. Start with simple tests (2-9). Then throw in a few tests with face cards. Finally add in tests with aces. Ping me if you need a hand getting started. Tests can be scary/hard to get going with.